### PR TITLE
rg-4: Fix common inputs conflicts

### PR DESCRIPTION
**Pull request by Valentyn13**

### CODE REVIEW

File: frontend/src/bundles/common/components/form-group/form-group.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { type ReactNode } from 'react';
+import { type FieldError } from 'react-hook-form';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+    width?: string;
+    error: FieldError | undefined;
```
Comment (nickolay-grechkin) [inline frontend/src/bundles/common/components/form-group/form-group.tsx:8]:
```suggestion
    error?: FieldError;
```

File: frontend/src/bundles/common/components/form-group/form-group.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+};
+
+const FormGroup = ({
+    error,
+    label,
+    children,
+    width = 'auto',
+}: Properties): JSX.Element => {
+    const errorMessage = error?.message;
+    return (
```
Comment (nickolay-grechkin) [inline frontend/src/bundles/common/components/form-group/form-group.tsx:20]: It is a common practice to add space before return statement to increase redability
```suggestion
    const errorMessage = error?.message;
    
    return (
```

File: frontend/src/bundles/common/components/input/input.tsx [hunk 3]
```diff
@@ -1,41 +1,26 @@
...
+    ({ hasError = false, width = 'auto', disabled, ...otherProperties }, reference) => (
+        <input
+            className={clsx(styles.input, {
+                [styles.input__error]: hasError,
+                [styles.input__disabled]: disabled,
+            })}
+            style={{ width }}
+            ref={reference}
+            type="text"
+            {...otherProperties}
```
Comment (nickolay-grechkin) [inline frontend/src/bundles/common/components/input/input.tsx:20]: Call spread operator before other props

File: frontend/src/bundles/common/components/input/input.tsx [hunk 4]
```diff
@@ -1,41 +1,26 @@
...
+        />
+    ),
+);
 
+Input.displayName = 'Input';
```
Comment (nickolay-grechkin) [inline frontend/src/bundles/common/components/input/input.tsx:25]: Why do we need this?

File: frontend/src/bundles/common/components/text-area/text-area.tsx [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+const TextArea = forwardRef<HTMLTextAreaElement,Properties>(
+    ({ hasError = false, width = 'auto', disabled, ...otherProperties }, reference) =>(
+    <textarea 
+        className={clsx(styles.textArea,{
+            [styles.textArea__error]: hasError,
+            [styles.textArea__disabled]: disabled
+        })}
+        style={{ width }}
+        ref={reference}
+        {...otherProperties}
```
Comment (nickolay-grechkin) [inline frontend/src/bundles/common/components/text-area/text-area.tsx:21]: Same as for above spread operator

**Review by nickolay-grechkin — CHANGES_REQUESTED** [review]

File: frontend/src/bundles/common/components/input/input.tsx (hunk 4 above)
Comment (Vitaliy-Kalenichenko) [reply frontend/src/bundles/common/components/input/input.tsx:25]: https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/display-name.md

**Review by Vitaliy-Kalenichenko — APPROVED** [review]

**Pull request merged**